### DISCUSSION TITLE
completion-window not opening when typing $this-> (in a .php file)

### DISCUSSION THREAD

**Discussion by dasraab** [hrsh7th/nvim-cmp#594] [Q&A]
Hi,

when i'm in a JS file and type "console." the completion window is opening showing me autocompletion for log, warn, debug etc... its just working fine.

if i type "$this->" in a .php file the opening of the completion-window is not triggered. it only get triggered if i type in one more char.

https://user-images.githubusercontent.com/1923919/144061735-b649c903-6dde-42e9-92ed-fe870ac6f071.mov

can anyone help or guide me into the right direction how to fix that (so that the completion gets triggered right after $this->)

thank you!

**Comment by hrsh7th** [comment]
I can't reproduce this.

**Reply by dasraab** [reply]
Found it. It seems to be a problem with [nvim-ts-autotag](https://github.com/windwp/nvim-ts-autotag). after disabling it everything works as expected. thanks for your great plugin!
